src/config/locales.md

```diff
@@ -9,8 +9,8 @@ $ locale -a
 ## Enabling locales
 
 To enable a certain locale, un-comment or add the relevant lines in
-`/etc/default/libc-locales` and [reconfigure](../xbps/packages/management.md)
-the `glibc-locales` package.
+`/etc/default/libc-locales` and reconfigure the `glibc-locales` package using
+[xbps-reconfigure(1)](https://man.voidlinux.org/xbps-reconfigure.1).
```

Comment:
The reason I like to link to the reconfiguration page is that it mentions the need for the `-f` flag.

Reply:
Ah, fair point. i'll change this back.